### DISCUSSION TITLE
Has ASReview been used in any peer-reviewed article?

### DISCUSSION THREAD

**Discussion by LHofstee** [asreview/asreview#558] [Q&A]
I would like to know whether there are peer-reviewed articles in which ASReview has been used.

**Comment by boer0107-zz** [comment]
Since ASReview is quite new, it will take some time before systematic reviews using ASReview will be published. One of the first published reviews using ASReview is this one:
van Haastrecht, M.; Yigit Ozkan, B.; Brinkhuis, M.; Spruit, M. Respite for SMEs: A Systematic Review of Socio-Technical Cybersecurity Metrics. Appl. Sci. 2021, 11, 6909. [https://doi.org/10.3390/app11156909](https://doi.org/10.3390/app11156909)

To find more examples, check our citations in Google Scholar:

 [An open source machine learning framework for efficient and transparent systematic reviews](https://scholar.google.nl/citations?view_op=view_citation&user=zclEqP8AAAAJ&citation_for_view=zclEqP8AAAAJ:vDZJ-YLwNdEC)

**Comment by boer0107-zz** [comment]
Update: we have compiled a [list of peer reviewed systematic reviews](https://www.zotero.org/groups/4597652/asreview_public/collections/IWU8ATMK/tags/peer_reviewed_article,systematic_review/collection) which made use of ASReview.
